From: unknown author <unknown.author@example.com>
Date: Sun, 19 Feb 2012 15:47:23 +0100
Subject: [PATCH] Fix ixp4xx_crypto compilation

This patch fixes following compile errors:

drivers/crypto/ixp4xx_crypto.c: In function 'ixp_module_init':
drivers/crypto/ixp4xx_crypto.c:1473:21: error: 'THIS_MODULE' undeclared (first use in this function)
drivers/crypto/ixp4xx_crypto.c:1473:21: note: each undeclared identifier is reported only once for each function it appears in
drivers/crypto/ixp4xx_crypto.c: At top level:
drivers/crypto/ixp4xx_crypto.c:1502:16: error: expected declaration specifiers or '...' before string constant
drivers/crypto/ixp4xx_crypto.c:1503:15: error: expected declaration specifiers or '...' before string constant
drivers/crypto/ixp4xx_crypto.c:1504:20: error: expected declaration specifiers or '...' before string constant
---
 drivers/crypto/ixp4xx_crypto.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/crypto/ixp4xx_crypto.c b/drivers/crypto/ixp4xx_crypto.c
index 4c20c5b..45704be 100644
--- a/drivers/crypto/ixp4xx_crypto.c
+++ b/drivers/crypto/ixp4xx_crypto.c
@@ -18,6 +18,7 @@
 #include <linux/interrupt.h>
 #include <linux/spinlock.h>
 #include <linux/gfp.h>
+#include <linux/module.h>
 
 #include <crypto/ctr.h>
 #include <crypto/des.h>
